src/users/user-router.js

```diff
@@ -2,18 +2,16 @@ const express = require('express');
 const userRouter = express.Router();
 
 const userManager = require('./user-manager.js');
+const verify = require('../auth/auth-manager.js').verify;
 
 // Create User.
 userRouter.post('/', (req, res) => {
   userManager.createUser(req.body, (err, id) => {
-    if (err) {
+    if (err || !id) {
       res.status(500).json(err).end();
```

Comment:
Wouldn't this status usually be client error (status 400), since this will typically be due to invalid parameters?